package/src/forms/UniversalForm.jsx

```diff
@@ -401,6 +416,14 @@ const componentForStep = (state) => {
         title: strings[type].done,
         Component: Success,
       };
+    case "password.showEmailSent":
+      return {
+        title: strings[type].password.checkEmail,
```

Comment:
This title might be a bit long. In similar places the title is "Check your email" and the message is "A link has been sent to ${email}." Since the user isn't expecting an email here maybe the message should be more like "You don't have a password set, click the link in your email to set your password" or similar. (Are we always going to send a password reset email, not a passwordless sign-on email?)

![Screenshot 2023-11-30 at 10 22 24 PM](https://github.com/userfront/toolkit/assets/17805905/6e3fc858-7fa2-4e74-8bde-612333caa3d8)